### ISSUE TITLE
 latex font missing

### ISSUE THREAD

**Issue by fightinggg** [next-theme/plugins#95]
<img width="200" alt="image-2021-06-22 19 36 19 180" src="https://user-images.githubusercontent.com/42932982/122917907-1f132880-d391-11eb-83e8-e7c4c8dbb451.png">
<img width="253" alt="image-2021-06-22 19 36 51 827" src="https://user-images.githubusercontent.com/42932982/122917983-32be8f00-d391-11eb-9805-74c7b0c3cec4.png">

latex font missing

**Comment by fightinggg** [comment]
_config.next.yml

math:
  # Default (false) will load mathjax / katex script on demand.
  # That is it only render those page which has `mathjax: true` in front-matter.
  # If you set it to true, it will load mathjax / katex srcipt EVERY PAGE.
  every_page: false

  mathjax:
    enable: true
    # Available values: none | ams | all
    tags: none

  katex:
    enable: false
    # See: https://github.com/KaTeX/KaTeX/tree/master/contrib/copy-tex
    copy_tex: false

**Comment by fightinggg** [comment]
`_config.next.yml`
```yml
math:
  # Default (false) will load mathjax / katex script on demand.
  # That is it only render those page which has `mathjax: true` in front-matter.
  # If you set it to true, it will load mathjax / katex srcipt EVERY PAGE.
  every_page: false

  mathjax:
    enable: true
    # Available values: none | ams | all
    tags: none

  katex:
    enable: false
    # See: https://github.com/KaTeX/KaTeX/tree/master/contrib/copy-tex
    copy_tex: false
```

**Comment by fightinggg** [comment]
when I add this code in `node_modules/@next-theme/plugins/index.js` It works 
```diff
  vendors.fontawesome_font = {
    name: '@fortawesome/fontawesome-free',
    file: 'webfonts'
  };
  vendors.katex_font = {
    name: 'katex',
    file: 'dist/fonts'
  };
+ vendors.mathjax_font = {
+   name: "mathjax",
+   file: "es5/output/chtml/fonts"
+ }
  for (const value of Object.values(vendors)) {
    const { data, error } = readFile(value);
    if (data) generator = generator.concat(data);
    if (error) errors.push(error);
  }
```

**Comment by stevenjoezhang** [comment]
Thanks for your feedback. Fixed in ed264ba746f32ac021a14524814ace7fdc427547
